### PR TITLE
chore: allow customising window handle safe area padding

### PR DESCRIPTION
**Pull request by junaid1460**
Situation: We should be able to use toolbar within sub view and that need not have this padding 

Solution: Let end dev customise this padding

## Pre-launch Checklist

- [ ] I have incremented the package version as appropriate and updated `CHANGELOG.md` with my changes 
- [ ] I have added/updated relevant documentation 
- [ ] I have run "optimize/organize imports" on all changed files
- [ ] I have addressed all analyzer warnings as best I could

### CODE REVIEW

**Comment by junaid1460** [comment]
Still a draft, I will write tests and let you know when it's ready.

File: lib/src/layout/toolbar/sliver_toolbar.dart [hunk 1]
```diff
@@ -41,6 +41,7 @@ class SliverToolBar extends StatefulWidget with Diagnosticable {
     this.floating = false,
     this.toolbarOpacity = 0.9,
     this.allowWallpaperTintingOverrides = true,
+    this.windowHandlePadding,
```
Comment (junaid1460) [inline lib/src/layout/toolbar/sliver_toolbar.dart:44]: safeAreaPadding?

**Comment by GroovinChip** [comment]
@junaid1460 I assume this aims to address #359?

**Comment by junaid1460** [comment]
> @junaid1460 I assume this aims to address #359?

No, I didn't look at it before this, I just tried this library and found this padding blocking my design goal, it was hardcoded.
Great if it solves that too. from what I read, it should.

**Comment by junaid1460** [comment]
I have made couple of more changes. un related to original change, pulldown button isn't customizable right now

**Comment by GroovinChip** [comment]
@junaid1460 are you still working on this?

**Comment by junaid1460** [comment]
> @junaid1460 are you still working on this?

Nope, I think we can close it or if you like to take over please do so. I have lost the context.

**Pull request closed without merging**